src/Extensions.ts

```diff
@@ -55,5 +55,27 @@ export const extensions: Extension[] = [
     supportedMarkets: {
       '1_USDC_0xc3d688B66703497DAA19211EEdff47f25384cdc3': null
     }
+  },
+  {
+    id: 'collateral_swap',
+    name: 'Collateral Swap',
+    description: 'Enables swapping collateral on open loan positions without the need to repay the loan, in a single transaction',
+    developer: 'Wido Labs',
+    links: {
+      github: 'https://github.com/widolabs/compound-collateral-extension-ui',
+      website: 'https://www.joinwido.com/'
+    },
+    permissions: {
+      sign: '*',
+    },
+    source: {
+      url: "https://wido-extension-template.vercel.app/embedded.html"
```

Comment:
Could we use an ipfs link instead? See the `comet_migrator` extension above for an example of how to configure it.

Reply:
Would it be acceptable to use our backend to serve the "Beta" version, and only deploy the final version to IPFS?

Please let us know if deploying to IPFS even for the Beta launch is necessary and we will do that.

Note: [In this post](https://www.comp.xyz/t/collateral-swap-extension-beta-launch-plan-feedback/4408), we proposed the Beta to run for 1 month, in which we would collect feedback. After incorporating the feedback we can launch the final version.

Reply:
That sounds reasonable, if the goal is iterate and make changes during the "Beta" phase.

Reply:
Just spoke with the team about this and we think using IPFS links is necessary. Even though most third-party devs like yourself that are taking the time to build extensions are probably very trustworthy, we want to remove that layer of trust altogether and make sure that whatever is approved as an extension can't be changed without another set of approvals.